lib/customers.js

```diff
@@ -238,13 +238,13 @@ function deleteEditedData (id, data) {
     'name'
   ]
   const filteredData = _.pick(defaults, _.mapKeys(_.snakeCase, data))
-  if (_.isEmpty(filteredData)) return getCustomerById(id)
+  if (_.isEmpty(filteredData)) return getCustomers({ id })
 
   const cs = new Pgp.helpers.ColumnSet(_.keys(filteredData),
     { table: 'edited_customer_data' })
   const update = Pgp.helpers.update(filteredData, cs)
   db.none(update)
-  return getCustomerById(id)
+  return getCustomers({ id })
```

Comment:
Let's leave everything as `getCustomerById()`, move this logic into the function instead of copy pasting everywhere.